### PR TITLE
test(dropdown): Adding tests for different anchors #1274

### PR DESCRIPTION
**Pull request by dafo**
Closes # .  

Additional information related to this pull request:
Adding tests for input, image and igx-tabs

### CODE REVIEW

File: projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts [hunk 1]
```diff
@@ -343,8 +349,78 @@ describe('IgxDropDown ', () => {
...
+        const button = fixture.debugElement.query(By.css('input')).nativeElement;
+        const list = fixture.componentInstance.dropdown;
+        const mockObj = jasmine.createSpyObj('mockEvt', ['stopPropagation', 'preventDefault']);
+        spyOn(list.onSelection, 'emit').and.callThrough();
+        spyOn(list.onClosed, 'emit').and.callThrough();
+        spyOn(fixture.componentInstance, 'onSelection');
+        expect(list).toBeDefined();
+        expect(list.items.length).toEqual(4);
+        button.click(mockObj);
+        fixture.whenStable().then(() => {
```
Comment (wnvko) [inline projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts]: As you are using faceasync, remove fixture when stable and use tick()

File: projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts [hunk 2]
```diff
@@ -343,8 +349,78 @@ describe('IgxDropDown ', () => {
...
+        const button = fixture.debugElement.query(By.css('img')).nativeElement;
+        const list = fixture.componentInstance.dropdown;
+        const mockObj = jasmine.createSpyObj('mockEvt', ['stopPropagation', 'preventDefault']);
+        spyOn(list.onSelection, 'emit').and.callThrough();
+        spyOn(list.onClosed, 'emit').and.callThrough();
+        spyOn(fixture.componentInstance, 'onSelection');
+        expect(list).toBeDefined();
+        expect(list.items.length).toEqual(4);
+        button.click(mockObj);
+        fixture.whenStable().then(() => {
```
Comment (wnvko) [inline projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts]: As you are using faceasync, remove fixture when stable and use tick()

File: projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts [hunk 3]
```diff
@@ -343,8 +349,78 @@ describe('IgxDropDown ', () => {
...
+    it('Should be able to implement to igx-tabs anchor', () => {
+        const fixture = TestBed.createComponent(IgxDropDownTabsTestComponent);
+        fixture.detectChanges();
+        const tabs = fixture.componentInstance.tabs;
+        const list = fixture.componentInstance.dropdown;
+        const mockObj = jasmine.createSpyObj('mockEvt', ['stopPropagation', 'preventDefault']);
+        expect(list).toBeDefined();
+        expect(list.items.length).toEqual(4);
+        tabs.tabs.toArray()[0].nativeTabItem.nativeElement.dispatchEvent(new Event('click', { bubbles: true }));
+        fixture.whenStable().then(() => {
```
Comment (wnvko) [inline projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts]: Switch to faceasync, remove fixture when stable and use tick()

**Comment by dafo** [comment]
tick(300); is for some zone.js issue - https://github.com/angular/zone.js/issues/845

**Pull request merged**